spec/fixtures/forms/2021_2022.json

```diff
@@ -1099,9 +1099,10 @@
           "label": "Declaration",
           "depends_on": [
             {
-              "household_characteristics": "completed",
-              "household_needs": "completed",
-              "property_information": "completed"
+              "armedforces": 1
```

Comment:
Why does this depend on armedforces being 1 or 3?

Reply:
This is just to pass the tests. I wanted to keep it dependent on something so that the behaviour still gets tested, and one of the things in the test set up was armed forces, so I made it dependent on that. No significance otherwise